physical/aws/security-groups/allow-peers.nix

```diff
@@ -3,7 +3,9 @@ with lib;
 let
   inherit (pkgs.globals) cardanoNodePort topology;
   inherit (topology) coreNodes relayNodes byronProxies;
-  peers = map (n: n.name) (coreNodes ++ relayNodes ++ byronProxies);
+  peers = map (n: n.name) (coreNodes ++ relayNodes ++ byronProxies)
```

Comment:
`builtins.concatLists [ coreNodes relayNodes byronProxies ]` is much faster if we have a lot of elements here.